backend/app/services/lessons/create.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Lessons
+  class Create < Core::Service
+    def call(params:)
+      @params = params
+      @course = Course.find_by(id: params[:course_id])
```

Comment:
We're not using `@course` outside this method so we could just go with local variable instead